relays/ethereum/src/main.rs

```diff
@@ -266,7 +264,16 @@ fn substrate_sync_params(matches: &clap::ArgMatches) -> Result<SubstrateSyncPara
 		eth_sign: ethereum_signing_params(matches)?,
 		metrics_params: metrics_params(matches)?,
 		instance: instance_params(matches)?,
-		sync_params: HeadersSyncParams::substrate_sync_default(),
+		sync_params: HeadersSyncParams {
+			max_future_headers_to_download: MAX_FUTURE_HEADERS_TO_DOWNLOAD,
+			max_headers_in_submitted_status: MAX_SUBMITTED_HEADERS,
+			// since we always have single Substrate header in separate Ethereum transaction,
```

Comment:
Shouldn't the comment be the other way around? I.e. "we always have only one Ethereum header in each Substrate transaction"?

Reply:
It is about `sub2eth`, so comment was correct some time ago. Now we may have up to 4 sub headers in single eth tx => it is obsolete